### PR TITLE
Persistence

### PR DESCRIPTION
**Pull request by ADmitri42**

### CODE REVIEW

File: cpp_mean_game/MeanGame.pxd [hunk 1]
```diff
@@ -10,8 +10,8 @@ cdef extern from "evolve.h":
         int size();
         double get_b();
         void set_b(double);
-        vector[int] get_field();
+        vector[char] get_field();
         void set_field(vector[int]);
```
Comment (ev-br) [inline cpp_mean_game/MeanGame.pxd:14]: char, too

File: cpp_mean_game/meangame.pyx [hunk 2]
```diff
@@ -29,7 +29,7 @@ cdef class MeanGamePy:
         cdef npy_intp dims[2]
         dims[0] = self._L
         dims[1] = self._L
-        return PyArray_SimpleNewFromData(2, dims, NPY_INT, self.c_game.get_field_pointer())
+        return PyArray_SimpleNewFromData(2, dims, NPY_UINT8, self.c_game.get_field_pointer())
```
Comment (ev-br) [inline cpp_mean_game/meangame.pyx:32]: Need to cimport?

**Comment by ev-br** [comment]
Monolithic designs like these are typically frowned upon. Best separate calculation of observables from evolution.

**Comment by ADmitri42** [comment]
I do understand that a monolithic design is really bad, but wouldn't it be costly to call a function to evaluate one step and then iterate over the field to calculate persistence?

**Pull request merged**